<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QSpinBox" name="spinBox_max_x">
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="Max_X">
     <property name="styleSheet">
      <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';

}

background-color : white;</string>
     </property>
     <property name="text">
      <string>Max X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_min_y">
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_back">
     <property name="text">
      <string>go back</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Min_Y">
     <property name="styleSheet">
      <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';

}

background-color : white;</string>
     </property>
     <property name="text">
      <string>Min Y</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Min_X">
     <property name="styleSheet">
      <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';

}

background-color : white;</string>
     </property>
     <property name="text">
      <string>Min X</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QCustomPlot" name="widget" native="true"/>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox_max_y">
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="Max_Y">
     <property name="styleSheet">
      <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';

}

background-color : white;</string>
     </property>
     <property name="text">
      <string>Max Y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton_graphic">
     <property name="text">
      <string>apply</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox_min_x">
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
